code/modules/mob/mob_movement.dm

```diff
@@ -542,3 +542,10 @@
 /// Can this mob move between z levels
 /mob/proc/canZMove(direction, turf/target)
 	return FALSE
+
+/mob/Move()
+	. = ..()
+	if(client)
+		for(var/sound/s in GLOB.allAttachedSounds)
```

Comment:
better var name, please

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names